layouts/shortcodes/ecosystem/registry/search-form.html

```diff
@@ -100,11 +111,174 @@
   <ul class="list-unstyled" id="search-results"></ul>
   <ul class="list-unstyled" id="default-body">
     {{ range $key, $value := $registry -}}
-      {{ $value = merge $value (dict "_key" $key) -}}
-      {{ partial "ecosystem/registry/entry" (dict "value" $value "languageNames" $languageNames) }}
+    {{ $value = merge $value (dict "_key" $key) -}}
+    {{ partial "ecosystem/registry/entry" (dict "value" $value "languageNames" $languageNames) }}
     {{ end -}}
   </ul>
+  <!-- Pagination Controls -->
+  <div id="pagination" class="text-center">
+    <button id="prev-page" class="btn btn-outline-secondary">Previous</button>
+    <div id="pagination-numbers" class="d-inline-block"></div>
+    <button id="next-page" class="btn btn-outline-secondary">Next</button>
+  </div>
+
+
+  <!-- total pages -->
+  <div id="pagination" class="text-center">
+    <div id="pagination-numbers" class="d-inline-block"></div>
+  </div>
+
+  <!-- Go to Page Section -->
+  <div class="text-center mt-3">
+    <input id="page-input" type="number" class="form-control d-inline-block w-auto" placeholder="Go to page" min="1"
+      max="${totalPages}">
+    <button class="btn btn-outline-primary" onclick="goToPage()">Go</button>
+    <p id="page-error" class="text-danger"></p>
+  </div>
```

Comment:
review https://getbootstrap.com/docs/5.3/forms/form-control/ to learn how you can make sure that the input box and the button are aligned. Right now they are off by a few pixels